backend/api/models.py

```diff
@@ -2,4 +2,9 @@
 
 from django.db import models
 
-# Create your models here.
+class Business(models.Model):
+    camis = models.CharField(max_length=255)
```

Comment:
If you're certain that it's a unique identifier for businesses, then yes. I've fallen into the trap of setting local id's for everything in the past and it just gets confusing. There is an option to set an attribute as the primary key for a model, and there is another option to disable the default primary key (`id` int autoincrement).

You'll need to implement both options, so just look them up in the documentation.